token-syncer/project.clj

```diff
@@ -0,0 +1,34 @@
+;;
+;;       Copyright (c) 2017 Two Sigma Investments, LP.
+;;       All Rights Reserved
+;;
+;;       THIS IS UNPUBLISHED PROPRIETARY SOURCE CODE OF
+;;       Two Sigma Investments, LP.
+;;
+;;       The copyright notice above does not evidence any
+;;       actual or intended publication of such source code.
+;;
+(defproject token-syncer "0.1.0-SNAPSHOT"
+  :aliases {"test" ["with-profile" "+test" "test"]}
```

Comment:
Seems like this happens automatically:

https://github.com/technomancy/leiningen/blob/master/doc/PROFILES.md#task-specific-profiles